types/stylelint/index.d.ts

```diff
@@ -1,6 +1,7 @@
 import type * as PostCSS from 'postcss';
 import type { GlobbyOptions } from 'globby';
 import type { cosmiconfig, TransformSync as CosmiconfigTransformSync } from 'cosmiconfig';
+import { Severity } from '../../lib/formatters/preprocessWarnings';
```

Comment:
Can you rename the import?
It's kinda confusing to have it twice in the same file.

see l72

Reply:
Fixed. I used the `stylelint.Severity` here instead of importing it from elsewhere.